src/res/translation/translation_fr_FR.ts

```diff
@@ -1367,17 +1367,17 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="250"/>
         <source>Language</source>
-        <translation>Langue</translation>
+        <translation type="unfinished">Langue</translation>
```

Comment:
The .ts files should not show up in this PR. The PR seems to revert those files to an earlier status.

Reply:
I think I made a mistake Yesterday.
I will solve that. Thanks

Reply:
solved 
5aa0168